### DISCUSSION TITLE
CoreShop Admin panel not loaded

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#13922] [Q&A / Support]
We have install the Coreshop on pimcore skeleton and all command executed successfully without any error but when we open the admin panel of the pimcore can see the error on console and admin panel is not loaded
We have followed this steps->  https://github.com/coreshop/CoreShop#readme

![image](https://user-images.githubusercontent.com/52573912/209683675-4dbe49fd-0a34-4f2c-b7f6-4bdaf03600e9.png)

we have tried with assets:install and cache:clear but still not working.

**Comment by dpfaffenbauer** [comment]
What CoreShop Version did you install?

**Reply by ghost** [reply]
composer require coreshop/core-shop:^3.0
